samples/generated/static-spa/package.json

```diff
@@ -0,0 +1,17 @@
+{
+  "name": "static-spa",
+  "private": true,
+  "version": "1.0.0",
+  "main": "index.js",
+  "license": "MIT",
```

Comment:
remove, probably just inherit from root level.

Reply:
these generated samples are all standalone modules, they can be copied and run outside this repo with no additional modifications. But maybe we don't want to include a license. We are not actually publishing the module, although a customer might use the code and publish it. It should be clear to them that they have the rights to take this code and modify it without legal problems.

Reply:
the pattern sample repos followed is to use `apache-2.0` in root level, and no `licence` field in each sub-sample folder, would be safe to follow that, imo.

like: https://github.com/okta/samples-js-vue/blob/master/okta-hosted-login/package.json

Reply:
OK, good catch. I did not actually realize we are using the Apache-2.0 license.